subprojects/shipkit/src/test/groovy/org/shipkit/internal/gradle/util/ProjectUtilTest.groovy

```diff
@@ -0,0 +1,31 @@
+package org.shipkit.internal.gradle.util
+
+import org.gradle.api.GradleException
+import org.gradle.api.Project
+import spock.lang.Specification
+
+class ProjectUtilTest extends Specification {
+
+    def "requireRootProject check passes for rootProject"() {
+        def project = Mock(Project)
+
+        when:
+        ProjectUtil.requireRootProject(project, ProjectUtilTest)
+
+        then:
+        noExceptionThrown()
+    }
+
+    def "requireRootProject fails for non-rootProject"() {
+        def project = Mock(Project)
```

Comment:
Minor: I suggest to make it an initialized field to make the test cleaner and eliminate small duplication.